schedule/functional/extra_tests_textmode.yaml

```diff
@@ -17,16 +17,64 @@ conditional_schedule:
                 - console/yast2_lan_device_settings
             x86_64:
                 - console/yast2_lan_device_settings
+    zypper:
+        DISTRI:
+            opensuse:
```

Comment:
wouldn't this mean that these test would only be executed for opensuse? I guess we don't really need them on sles, or do we?

Reply:
I tried to resemble what the original schedule was doing.